charters/DataProcessingService/DataProcessingPipelines.md

```diff
@@ -0,0 +1,70 @@
+
+## Definitions
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project, but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports
+__Onboarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay 
+__Standardized Pipeline:__ An official pipeline of the HCA project 
+
+## In-scope
+* Scoping or review of scoping for new pipelines  
+* Creating all standardized pipelines for scientific data 
+* Reviewing, on-boarding, and potentially creating community pipelines  
+* Reviewing community tools for use in data processing pipelines 
+* Creating tools that support the creation of community pipelines - may include software, documentation or support
+* Updating pipelines ran in the HCA DCP for optimization of cost, time, or scientific benchmarks  
+* Establishing standard benchmarking used to evaluate data processing pipelines  
+* Investigating and identifying data for pipeline benchmarking
+* Evaluating and adopting file formats and associated standards for outputs of pipelines, ensuring the data is accessible with standard tools and interoperable with downstream processes 
+* Establishing and operating automated practices around scientific testing of pipelines
+* Performing outreach to users on the usability of data in downstream applications 
+* Performing outreach to developers on pipeline development in the HCA DCP 
+* Suggesting, but not selection of, new standardized pipelines for the HCA DCP 
+* Working with the community to provide feedback on metadata standards for raw data in the HCA DCP to provide the required information to enable processing of the raw data.
+* Working with the metadata team and data consumers to ensure the metadata provided about pipelines and the analysis results meet the needs of downstream users
+
+### Scientific "guardrails"
+
+* Initial versions of standardized pipelines are reviewed by the AWG.  
+* Changes to standardized pipelines that affect the scientific output are reviewed by the AWG, engineering updates that do not affect the scientific output are not reviewed by the AWG.  
```

Comment:
It could be a good idea to define what we mean by "updates that affect the scientific output".

Reply:
I used the word "changes" instead of "updates" that is more restricted in interpretation (hopefully).